src/main/java/com/github/pascalgn/jiracli/Jiracli.java

```diff
@@ -100,6 +106,15 @@ public static void main(String[] args) {
         map.put(Option.GUI, list.contains("-g") || list.contains("--gui"));
         map.put(Option.VERSION, list.contains("-V") || list.contains("--version"));
         list.removeAll(Arrays.asList("-h", "--help", "-c", "--console", "-g", "--gui", "-V", "--version"));
+        String line = null;
+        int commandIndex = list.indexOf("--command") + 1;
+        if (commandIndex > 0 && (list.size() > commandIndex) ) {
+            line = list.get(commandIndex);
+            map.put(Option.CMD_MODE, Boolean.TRUE);
+            map.put(Option.CMD_LINE, line);
```

Comment:
I think we don't need both, `CMD_LINE` could be enough, so we could check if `map.get(Option.CMD_LINE) != null` and don't need CMD_MODE anymore

Reply:
Agreed. Removed CMD_MODE